deployments/helm/cvmfs-csi/templates/cvmfs-repo-storageclass.yaml

```diff
@@ -0,0 +1,12 @@
+{{- range $repo := .Values.specificRepositoryStorageClasses -}}
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ $repo.name }}
+  labels:
+    {{- include "cvmfs-csi.common.metaLabels" .  | nindent 4 }}
+provisioner: {{ .Values.csiDriverName }}
```

Comment:
Can you please check this? This won't compile because `.` now refers to the range-loop context.

```patch
diff --git a/deployments/helm/cvmfs-csi/templates/cvmfs-repo-storageclass.yaml b/deployments/helm/cvmfs-csi/templates/cvmfs-repo-storageclass.yaml
index c9a687e9..a5a1f75c 100644
--- a/deployments/helm/cvmfs-csi/templates/cvmfs-repo-storageclass.yaml
+++ b/deployments/helm/cvmfs-csi/templates/cvmfs-repo-storageclass.yaml
@@ -5,8 +5,8 @@ kind: StorageClass
 metadata:
   name: {{ $repo.name }}
   labels:
-    {{- include "cvmfs-csi.common.metaLabels" .  | nindent 4 }}
-provisioner: {{ .Values.csiDriverName }}
+    {{- include "cvmfs-csi.common.metaLabels" $  | nindent 4 }}
+provisioner: {{ $.Values.csiDriverName }}
 parameters:
   repository: {{ $repo.repository }}
 {{ end }}
```

... I don't see the button for review suggestion, sorry. You can try the patch above.

Reply:
In theory I've got this sorted.

Reply:
Last bit missing is the `$.` in the `provisioner` field -- `$.Values` :)